geoutils/__init__.py

```diff
@@ -2,10 +2,12 @@
 GeoUtils is a Python package for the analysis of geospatial data.
 """
 
-from geoutils import examples, pointcloud, projtools, raster, vector  # noqa
+from geoutils import examples, projtools  # noqa
```

Comment:
so we don't want to load the raster submodule explicitely anymore? Is there nothing else to get apart from the Raster and Mask classes? (I'm not fully familiar with the new structure yet)